app/src/main/java/org/mozilla/focus/utils/DialogUtils.java

```diff
@@ -221,8 +221,8 @@ public static void showDefaultSettingNotification(Context context) {
         final PendingIntent openRocketPending = PendingIntent.getBroadcast(context, REQUEST_DEFAULT_CLICK, openDefaultBrowserSetting,
                 PendingIntent.FLAG_ONE_SHOT);
 
-        final String title = context.getString(R.string.app_name);
-        final String content = context.getString(R.string.preference_default_browser) + "?\uD83D\uDE4C";
+        final String title = context.getString(R.string.preference_default_browser) + "?\uD83D\uDE0A";
+        final String content = context.getString(R.string.notification_default_browser);
```

Comment:
This string is too long on my Pixel XL, we might need BigTextStyle?

Reply:
After discussion, we don't want this string for now (will be added in 2.0 with new string)